content/en/blog/signal-boosting.md

```diff
@@ -0,0 +1,290 @@
+---
+title: Signal Boosting
+date: 2022-09-24
+---
+
+The new release of Preact Signals brings significant performance updates to the underpinnings of the reactive system. Read on to learn what kinds of tricks we employed to make this happen.
```

Comment:
```suggestion
The new release of Preact Signals brings significant performance updates to the foundations of the reactive system. Read on to learn what kinds of tricks we employed to make this happen.
```

Wording suggestion, could be me not being familiar with "underpinngs" as expression so feel free to ignore

Reply:
Thanks, that's good 👍